pycbc/io/hdf.py

```diff
@@ -648,7 +648,8 @@ def get_column(self, cname):
         # If the mask accesses few enough elements then directly use it
         # This can be slower than reading in all the elements if most of them
         # will be read.
-        if self.mask is not None and (self.mask.sum() < len(self.mask) * MFRAC):
+        if self.mask is not None and not isinstance(self.mask, list) and \
```

Comment:
I don't think this change should be made. In fact a list *can* work in this way, it just must be ascending order. This would revert some of the functionality needed to keep the memory useage under control.

Reply:
Lists don't have a `sum` attribute!

However if the list is `[100,2000, 130000]` doing a `sum(list)` is not going to give the expected result! Probably we want to replace `self.mask.sum()` with "What is the actual length of the list"

Reply:
Yeah, that's a bug. Honestly, probably best to just put an or in and go here if it's a list always.

Reply:
Agreed, changed as suggested.